src/Core/global_functions.php

```diff
@@ -44,7 +44,12 @@ function rtr($path)
      */
     function which($command)
     {
-        exec(sprintf('which %s 2>&1', $command), $path, $exitCode);
+        if (stristr(PHP_OS, 'WIN')) {
+            exec(sprintf('where %s 2>&1', $command), $path, $exitCode);
+            $path = array_map('escapeshellarg', $path);
```

Comment:
What happens if the `escapeshellarg()` method is called in a unix environment?

Reply:
I don't know, that's why it will run only if PHP_OS is containing WIN